api/models/action-type.js

```diff
@@ -1,6 +1,6 @@
 const associate = ({ ActionType, UserAction }) => {
-  ActionType.belongsTo(UserAction, {
-    foreignKey: 'actionTypeId',
+  ActionType.hasMany(UserAction, {
+    foreignKey: 'actionId',
```

Comment:
I think `actionTypeId` is clearer here, since it is the FK to the `ActionType` table, right?

<img width="694" alt="screen shot 2018-01-02 at 3 49 39 pm" src="https://user-images.githubusercontent.com/697848/34501415-83a16a04-efd4-11e7-9d44-9eb4d91b7a6e.png">

Reply:
I don't know if its clearer, but IIRC it straight up doesn't work without this foreignKey. I think its less an issue of correct SQL and more appeasing sequelize's API.

Reply:
oh, like the string `'actionTypeId'` doesn't work, while `'actionId'` does?

Reply:
oh I see what you're saying, I guess we can go back and change it, I think we'll have to write another migration though.

Reply:
actionTypeID is the automatic foreignKey prop that sequelize assigns under the hood. I think I set it to this value when I was trying to get these associations to behave properly. It seemed clearer than some of our current foreign key naming conventions.